### PR TITLE
Move contrib guidelines to CONTRIBUTING.md, shorten README

### PR DESCRIPTION
**Pull request by bsmth**
This PR makes the project README shorter and includes only build instructions to get running.

Full contribution guidelines are covered in `CONTRIBUTING.md`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,562 +1,47 @@
...
-[content of MDN Web Docs](https://github.com/mdn/content), which is
-hosted within the [MDN Organization](https://github.com/mdn) on GitHub.
+MDN Web Docs is an open-source, collaborative project that documents web technologies including CSS, HTML, JavaScript, and Web APIs.
+Alongside detailed reference documentation, we provide extensive learning resources for students and beginners getting started with web development.
 
-> **Note:** Just want to find a task and jump in?
-> See [Getting started on MDN](https://developer.mozilla.org/en-US/docs/MDN/Community/Contributing/Getting_started)
-> for an overview of how to join, and [Contributing to MDN](https://developer.mozilla.org/en-US/docs/MDN/Contribute)
-> for a filtered list of tasks.
+### MDN's mission 🙌
```
Comment (Josh-Cena) [inline README.md:8]: There's no H2 heading; also IMO we should use less emojis in titles

Comment (bsmth) [reply README.md]: Headings updated in https://github.com/mdn/content/pull/21361/commits/aad890e91531f0732496cb54a7440b1c8c1e8d10

I think the emojis make the README (and project) look a bit more approachable, but happy to leave this one open / change according to feedback.

Comment (bsmth) [reply README.md]: I understand. Let me remove for now in https://github.com/mdn/content/pull/21361/commits/0361f6d816552d740bbba39d9cd6117e4b20ce89

**Review by dipikabh — APPROVED** [review]
LGTM! Thank you 🙌

**Pull request merged**